pom.xml

```diff
@@ -1636,6 +1636,36 @@
       </activation>
     </profile>
 
+    <profile>
+      <id>spark3.2.0</id>
```

Comment:
@nsivabalan sorry, just catching up with this PR.

Didn't we discuss that we should add capability to build against any Spark version (generic profile)